lib/package.json

```diff
@@ -35,9 +35,11 @@
     ]
   },
   "dependencies": {
-    "noble": "^1.9.0"
+    "noble": "^1.9.0",
+    "noble-mac": "git+https://github.com/Timeular/noble-mac.git"
```

Comment:
do you think is it possible to use npm package? I had problems before using a direct link to github.

Reply:
At the moment there is still no npm package available (see issue https://github.com/Timeular/noble-mac/issues/19) 😞 
I could try to switch to this deprecated version, if you think this would make sense (https://www.npmjs.com/package/@s524797336/noble-mac)

Reply:
Ok, then, can you create an issue for this, so we can track it. Thanks!